examples/plot_clip.py

```diff
@@ -0,0 +1,156 @@
+#
+# The example below walks you through a typical workflow for clipping one
+# vector data file to the shape of another. Both vector data files must be
+# opened with GeoPandas as GeoDataFrames and be in the same Coordinate
+# Reference System (CRS) for the ``clip_shp()`` function in EarthPy to work.
+#
+# This example uses Polygons, a Line, and Points made with shapely and then
+# turned into GeoDataframes.
+#
+# .. note::
+#    The object to be clipped will be clipped to the full extent of the clip
+#    object. If there are multiple polygons in clip object, the input data will
+#    be clipped to the total boundary of all polygons in clip object.
+
+###############################################################################
+# Import Packages
+# ---------------
+#
+# To begin, import the needed packages. You will primarily use EarthPy's clip
+# utility alongside GeoPandas.
+
+import numpy as np
+import matplotlib.pyplot as plt
+import geopandas as gpd
+from shapely.geometry import Polygon, LineString, Point
+import geopandas.clip as gc
+
+###############################################################################
+# Create Example Data
+# -------------------
+#
+# Once the packages have been imported, you need to create the shapes to clip.
+# You need to make two polygons, one line, and one point feature with shapely,
+# and then open those shapes up with GeoPandas.
+
+polygon1 = Polygon([(0, 0), (0, 10), (10, 10), (10, 0), (0, 0)])
+polygon2 = Polygon([(-5, -5), (-5, 5), (5, 5), (5, -5), (-5, -5)])
+line = LineString([(3, 4), (5, 7), (12, 2), (10, 5), (9, 7.5)])
+pts = np.array([[2, 2], [3, 4], [9, 8]])
+
+###############################################################################
+# Open Files with GeoPandas and Reproject the Data
+# -------------------------------------------------
+#
+# Open the data files to as GeoDataFrames using GeoPandas.
+#
+# .. note::
+#    Recall that the data must be in the same CRS in order to use the
+#    ``clip()`` function. If the data are not in the same CRS, be sure to use
+#    the ``to_crs()`` function from GeoPandas to match the projects between the
+#    two objects, as shown below. In this example, since you make all of the data,
+#    you don't have to change the CRS.
+
+# Now that since you have all of the shapes created, you can open them with GeoPandas
+poly_gdf1 = gpd.GeoDataFrame(
+    [1], geometry=[polygon1], crs={"init": "epsg:4326"}
+)
+poly_gdf2 = gpd.GeoDataFrame(
+    [1], geometry=[polygon2], crs={"init": "epsg:4326"}
+)
+line_gdf = gpd.GeoDataFrame([1], geometry=[line], crs={"init": "epsg:4326"})
+points_gdf = gpd.GeoDataFrame(
+    [Point(xy) for xy in pts], columns=["geometry"], crs={"init": "epsg:4326"}
+)
+
+###############################################################################
+# The plot below shows all of the data before it has been clipped. Notice that
+# the ``.boundary`` method for a GeoPandas object is used to plot the
```

Comment:
boundary may be an attribute i think?? but regardless you dont need any steps here either. please just have a section title that says - plot the unclipped data.